packages/core/useScriptTag/index.ts

```diff
@@ -171,9 +177,20 @@ export function useScriptTag(
     if (scriptTag.value)
       scriptTag.value = null
 
-    const el = document.querySelector<HTMLScriptElement>(`script[src="${resolveUnref(src)}"]`)
-    if (el)
-      document.head.removeChild(el)
```

Comment:
Instead of querySelector, maybe we could keep the reference of `el` from creation.